packages/eslint-plugin/src/rules/jsx-pragma/__tests__/rule.test.ts

```diff
@@ -581,15 +739,15 @@ import * as React from 'react';
     },
     {
       name: 'should consider libraries in importSources to be Compiled imports',
-      code: `
+      code: outdent`
         import { css } from '@atlaskit/css';
 
         const styles = css({ display: 'block' });
         <div css={styles} />
       `,
-      output: `
-        /** @jsxImportSource @compiled/react */
-import { css } from '@atlaskit/css';
+      output: outdent`
+        /** @jsxImportSource @atlaskit/css */
```

Comment:
This one is a **change**, but I'm _pretty certain_ the name of the test implied this should work?

Reply:
that looks reasonable to me

assuming that you've done a quick check that `/** @jsxImportSource @atlaskit/css */` does work

Reply:
Actually, no, we use `classic` mostly—any good place to test (feel free to DM)?